recipes/ri/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+sed -i.bak '2d' makefile
```

Comment:
I think this line can be removed (also from bld.bat), given we are using makefile.patch now.